js/screens/Profile/ProfileScreen.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import { Button, AsyncStorage } from 'react-native';
+import PropTypes from 'prop-types';
+
+import { colors } from '../../config/styles';
+
+import Profile from './Profile';
+
+class ProfileScreen extends Component {
+  static propTypes = {
+    navigation: PropTypes.object
```

Comment:
Please add default props.

Also, please use either ES6 or ES7 to define propTypes. Don't use both.